backend/src/agents/validator_agent.py

```diff
@@ -0,0 +1,42 @@
+from src.utils import call_model
+from src.agents import Agent, agent_metadata
+
+validator_prompt = """
+You are an expert validator. You can help with validating the answers to the tasks with just the information provided.
+
+You're entire purpose is to return a boolean value to indicate if the answer has fulfilled the task.
+
+You will be passed a task and an answer. You need to determine if the answer is correct or not.
+
+e.g.
+Task: What is 2 + 2?
+Answer: 4
+Response: True
+
+Task: What is 2 + 2?
+Answer: 5
+Response: False
+
+Task: Find all spending transactions last month on Amazon.
+Answer: Last month you spend £64.21 on Amazon
+Response: True
+
+Task: Find all spending transactions last month on Amazon.
+Answer: Last month you spend £64.21 on Spotify
+Response: False
+Reasoning: The answer is for Spotify not Amazon.

```

Comment:
```
Reasoning: The answer is for Spotify not Amazon.
```

Do we need this line? It might confuse the LLM to return a response with a Reasoning field. Saying this with no example of it doing so but wanted to raise the concern all the same - if it performs as well without I personally think we should remove it

Reply:
hmm I think it helped whilst I was testing. I think the next couple of lines are enough to make sure it only returns the boolean